src/main/java/teammates/common/util/Config.java

```diff
@@ -74,6 +80,8 @@
         } catch (IOException e) {
             Assumption.fail(TeammatesException.toStringWithStackTrace(e));
         }
+        APP_ID = properties.getProperty("app.id");
+        APP_VERSION = properties.getProperty("app.version");
```

Comment:
The change `app.version` does not replace `-` with `.` (e.g., "6-0-0" -> "6.0.0") while the old one does. Is it ok to do this?

Reply:
Strictly speaking it's ok, but I guess I'll change back to keep the status quo.